modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/CheckpointWorkflowTest.java

```diff
@@ -181,13 +171,13 @@ void testMarkCheckpointBegin() throws Exception {
 
         CheckpointReadWriteLock readWriteLock = newReadWriteLock(log);
 
-        List<FullPageId> dirtyPages = List.of(new FullPageId(0, 0), new FullPageId(1, 0), new FullPageId(2, 0));
+        List<FullPageId> dirtyPages = List.of(of(0, 0, 0), of(0, 0, 1), of(0, 0, 2));
```

Comment:
`of()` method name does not seem the best choice as it gives the reader no clue about what kind of objects it constructs. Could we rename it?